skills_ml/ontologies/esco.py

```diff
@@ -0,0 +1,81 @@
+from .base import Competency, Occupation, CompetencyOntology
+import logging
+import csv
+import requests
+
+lang = '&language=en'
+api_tax = 'https://ec.europa.eu/esco/api/resource/taxonomy?uri=http://data.europa.eu/esco/concept-scheme/isco&language=en'
+
+
+class Esco(CompetencyOntology):
+    def __init__(self):
+        super().__init__()
+        self.is_built = False
+        self.competency_framework.name = 'esco_skills'
+        self.competency_framework.description = 'ESCO Knowledge, Skills/Competencies'
+        self._build()
+    
+    def _build(self):
+        if not self.is_built:
+            reqTax = requests.get(api_tax)
+            getConcepts = reqTax.json()['_links']['hasTopConcept']
+            for concept in getConcepts:
+                _conceptProcessing(concept['href'])
+        else:
+            logging.warning('ESCO Ontology is already built!')
+
+    def _conceptProcessing(href):
+        print href
```

Comment:
skills-ml requires Python3.6, this print won't work on that. I'd test this using a Python3.6 virtualenv as described in the README